datastore/data-loader/client_knmi_station.py

```diff
@@ -29,6 +29,32 @@
 regex_time_period = re.compile(r"(\d+) (Hours|Min)", re.IGNORECASE)
 
 
+def convert_standard_names_to_cf(standard_name):
+    standard_name_mapping = {
+        "cloud_cover": "cloud_area_fraction",
+        "total_downwelling_shortwave_flux_in_air": "surface_downwelling_shortwave_flux_in_air",
+        "precipitation_rate": "rainfall_rate",
+        "air_pressure_at_sea_level": "air_pressure_at_mean_sea_level",
+    }
+    return standard_name_mapping.get(standard_name, standard_name)
+
+
+# NOTE: Only units are converted currently, not values.
```

Comment:
Definetly something to be aware of, as we know have "unlikely" data.

Reply:
Yes, this is something I would have liked to avoid, but it lessens the burden to maintain two different `data-loaders` so that they can be used with same comparison data.